lib/fs/fileinfo_unix.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// +build !windows
+
+package fs
+
+import "syscall"
```

Comment:
It hardly matters here, but shouldn't syscall generally be avoided in favor of golang.org/x/sys/... packages?

Reply:
Not in this case, because we need to match on what is actually returned. :)